### PR TITLE
Lvnodcdxi 63

### PR DESCRIPTION
**Pull request by skiffua**
Made configuration for enzyme, tested one component "canvasEqualiser"

### CODE REVIEW

**Review by GeFeSTTS — DISMISSED** [review]

**Review by Kindest13 — DISMISSED** [review]

File: src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Enzyme, {
+  shallow,
+} from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import CanvasEqualizer from './canvasEqualizer';
+
+Enzyme.configure({
```
Comment (tonypntsrn) [inline src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js]: Don’t you
Add this config, to config file?

File: src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+});
+
+describe('Grafic equaliser', () => {
+  const props = {
+    width: 150,
+    height: 100,
+    getCanvasEl: () => {},
+  };
+
+  it('should RENDER CANVAS ELEMENT', () => {
```
Comment (tonypntsrn) [inline src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js]: Why all words are uppercase?

File: src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js (hunk 1 above)
Comment (skiffua) [reply src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js]: ok

File: src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js (hunk 2 above)
Comment (skiffua) [reply src/client/components/equalizer/canvasEqualizer/canvasEqualizer.test.js]: done

**Review by tonypntsrn — APPROVED** [review]

**Review by Kindest13 — APPROVED** [review]

**Pull request merged**